portfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="portfolio.css">
+</head>
+
+<body>
+    <section class="background">
+        <div class="toolbar">
+            <div class="welcome">Welcome</div>
+            <img class="menu" src="./assets/menu.svg" alt="menu">
+        </div>
+        <!-- <div style="display: flex; justify-content: center;">
+            <p class="margin-10px" style="cursor: pointer;">Portfolio</p>
+            <p class="margin-10px" style="cursor: pointer;">About</p>
+            <p class="margin-10px" style="cursor: pointer;">Contact</p>
+        </div> -->
+        <p class="intro primary-color">Hey there. I’m Rex.</p>
+        <p class="intro secondary-color">I am a full stack developer.</p>
+        <p> I can help you build a product , feature or website.</p>
+        <p> Look through some of my work and experience!</p>
+        <p> If you like what you see and have a project you need coded, don’t hestiate to contact me. 😉</p>
+        <div class="center">
+            <a href="">
+                <img class="margin-10px" src="./assets/twitter.svg" alt="twitter">
+            </a>
+            <a href="">
+                <img class="margin-10px" src="./assets/linkedin.svg" alt="linkedin">
+            </a>
+            <a href="">
+                <img class="margin-10px" src="./assets/medium.svg" alt="medium">
+            </a>
+            <a href="">
+                <img class="margin-10px" src="./assets/github.svg" alt="github">
+            </a>
+            <a href="">
+                <img class="margin-10px" src="./assets/angellist.svg" alt="angellist">
+            </a>
+        </div>
+        <!-- <div class="center margin-top-25px">
+            <img style="cursor: pointer;" src="./assets/down-arrow.svg" alt="angellist">
+        </div>
+        <header style="padding-top: 25px;">My Recent Works</header style="padding-top: 25px;"> -->
+
+            <!-- <div class="card">
+                card
+            </div> -->
```

Comment:
- [ ] Remove the commented code from your project as it makes code hard to read.